.goreleaser.yml

```diff
@@ -0,0 +1,24 @@
+before:
+  hooks:
+    - make validate
+builds:
+  - goos:
+      - linux
+    goarch:
+      - amd64
+    ldflags:
+      - -s -w -X github.com/code-ready/machine-driver-libvirt/pkg/libvirt.DriverVersion={{.Version}}
+    main: ./cmd/machine-driver-libvirt/main.go
+archives:
+  - format: binary
+    name_template: "{{ .ProjectName }}"
+checksum:
+  name_template: 'checksums.txt'
```

Comment:
I'd go with `sha256sum.txt` + `algorithm: sha256` as one does not need to guess what kind of hash was used.

Reply:
done